setup.py

```diff
@@ -12,7 +12,7 @@
 EMAIL = "sumeyyedilaradogan@gmail.com"
 AUTHOR = "Sddilora"
 REQUIRES_PYTHON = ">=3.6.0"
-VERSION = "1.0.2"
+VERSION = "1.0.3"
```

Comment:
This seems like a minor release as per [Semantic Versioning](https://semver.org/). You might want to upgrade the version to 1.1.0?

Reply:
I was planning to make this change after adding the usage of ORM to the documentation and implementing the rendering of MD files.

Reply:
Implementing the MD templating would make another minor version if you ask me .) If you're planning to update the documentation as well, we could merge this pr after you made that change. Or upgrade to 1.1.1 after documentation is added.

Reply:
ok I will handle that. Thanks for your suggestion.